### PR TITLE
Ephys task1a dev

### PR DESCRIPTION
**Pull request by corbennett**

### CODE REVIEW

**Comment by corbennett** [comment]
Hey Chris,
I think I made a mess of this. Can you point me in the right direction? Does this look ok?

File: regimen.yml [hunk 1]
```diff
@@ -430,7 +432,7 @@ stages:
     script: *replay_script
     script_md5: *replay_script_md5
     parameters:
-      <<: *replay_a_parameters
+      <<: *replay_b_parameters
```
Comment (mochic) [inline regimen.yml:439]: is this intentional?

**Review by mochic — CHANGES_REQUESTED** [review]
Everything looks good except for the commented params which i'm worried about being a typo.

File: regimen.yml (hunk 1 above)
Comment (corbennett) [reply regimen.yml:439]: Yeah. This fixes an old bug I happened to spot. This is a 'B' stage so it should point to the b_parameters.

**Comment by corbennett** [comment]
@mochic I forgot to add the opto params. But everything should be good now.

c

**Pull request merged**